tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/method/TeamMailboxMemberGetMethod.scala

```diff
@@ -41,31 +41,19 @@ class TeamMailboxMemberGetMethod @Inject()(val teamMailboxRepository: TeamMailbo
         methodCallId = invocation.invocation.methodCallId))
       .map(InvocationWithContext(_, invocation.processingContext))
 
-  private def getTeamMailboxMemberResponse(username: Username, request: TeamMailboxMemberGetRequest): SMono[TeamMailboxMemberGetResponse] = {
+  private def getTeamMailboxMemberResponse(username: Username, request: TeamMailboxMemberGetRequest): SMono[TeamMailboxMemberGetResponse] =
     request.ids match {
-      case None => getMembersOfAllTeamMailboxes(username, request.accountId)
-      case Some(ids) => getMembersOfSpecificTeamMailboxes(username, request.accountId, ids)
```

Comment:
Why changing this? If anything I think it's harder to read now

Reply:
It is suggestion from Tung. Fixed